devtools/dump_devinfo.py

```diff
@@ -477,6 +490,38 @@ def format_exception(e):
     return exception_str
 
 
+async def _make_final_calls(
+    protocol: SmartProtocol,
+    calls: list[SmartCall],
+    name: str,
+    batch_size: int,
+    *,
+    child_device_id: str,
+) -> dict[str, dict]:
```

Comment:
docstring what it means that these are final. it retests the known-to-work method calls, first in batches (if supported) and then one by one?

Reply:
Done